ckks/bootstrapp_test.go

```diff
@@ -12,12 +13,20 @@ import (
 
 func TestBootstrapp(t *testing.T) {
 
+	if !*testBootstrapping {
+		t.Skip("skipping bootstrapping test")
+	}
+
+	if runtime.GOARCH == "wasm" {
+		t.Skip("skipping bootstrapping tests for GOARCH=wasm")
+	}
+
 	rand.Seed(time.Now().UnixNano())
 
 	var err error
 	var testContext = new(testParams)
 
-	paramSet := uint64(0)
+	paramSet := uint64(1)
```

Comment:
We don't want to test the smallest set ?

Reply:
The -short flag will tune down N and n, regardless of the set, so it doesn't matter what is tested.